src/transformStateWithClones.js

```diff
@@ -7,7 +7,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
```

Comment:
It is better to avoid naming variables 'result'. It would be great to specify what exactly the result is

Reply:
const clone = [];